packages/components/Avatar/src/useAvatar.ts

```diff
@@ -137,7 +138,8 @@ export const removeRedundantCharacters = (name: string): string[] => {
   }
 
   const MAX_NAME_LENGTH = 100;
-  const _name = name.length > MAX_NAME_LENGTH ? name.trim().substring(0, MAX_NAME_LENGTH) : name;
+  let _name = name.length > MAX_NAME_LENGTH ? name.trim().substring(0, MAX_NAME_LENGTH) : name;
+  _name = isMultiWordName(_name) ? removeMultiWordTitles(_name) : _name;
```

Comment:
If removeMultiWordTitles checks for isMultiWordName, we don't need to check it here and do duplicate work

Reply:
I left it here and removed from the function